report/finished/chapters/modeling.tex

```diff
@@ -1,10 +1,60 @@
 \section*{Robotic arm}
 \subsection*{Forward kinematics}
 \subsection*{Inverse kinematics}
+
+
+%%%%%%%%%%%%%%%%%%
+
 \section*{Base}
 
```

Comment:
The equation references dont work for me when i compile, also the picture dont load. I've sync with REVIEW_BaseModeling.